
std_msgs/Header header
geometry_msgs/Vector3 position_estimate
geometry_msgs/Vector3 velocity_estimate
geometry_msgs/Vector3 omega_estimate
float64 roll_estimate
float64 pitch_estimate
float64 yaw_estimate
float64 mass_estimate
float64 mass_covariance
float64 Jxy_estimate
float64 Jxy_covariance
float64 Jz_estimate
float64 Jz_covariance
float64 minimal_singular_value
float64 force_torque_projection
float64 O_determinant
float64 com_z
float64 com_z_covariance
geometry_msgs/Vector3 com_estimate
geometry_msgs/Vector3 torque_com



